tests/UnitTests/Acheve.TestHost/Routing/TestServerExtensionsTests.cs

```diff
@@ -448,7 +452,7 @@ public void create_valid_request_using_from_body_complex_arguments()
         }
 
         [Fact]
-        public void create_valid_request_using_from_body_complex_arguments_and_primitive_query_parameters()
+        public async Task create_valid_request_using_from_body_complex_arguments_and_primitive_query_parameters()
```

Comment:
If you add a option to include or not the content in RequestBuilder probably you need to split this on diferent test cases for both options